src/LookingGlassWebXRPolyfill.ts

```diff
@@ -20,20 +20,43 @@ import WebXRPolyfill from "@lookingglass/webxr-polyfill/src/WebXRPolyfill"
 import { getLookingGlassConfig, updateLookingGlassConfig, ViewControlArgs } from "./LookingGlassConfig"
 import LookingGlassXRDevice from "./LookingGlassXRDevice"
 import LookingGlassXRWebGLLayer from "./LookingGlassXRWebGLLayer"
+import * as HoloPlayCore from "holoplay-core"
 
 export class LookingGlassWebXRPolyfill extends WebXRPolyfill {
 	private vrButton: HTMLButtonElement | undefined
-	public device: LookingGlassXRDevice
+	public device: LookingGlassXRDevice | undefined
 	/** true when previewing on Looking Glass */
 	public isPresenting: boolean = false
 
 	constructor(cfg?: Partial<ViewControlArgs>) {
 		super()
-
 		// Init the configuration
 		updateLookingGlassConfig(cfg)
+		this.loadPolyfill()
```

Comment:
currently this code here adds the base for Looking Glass device detection. This does currently require some changes to how XR is initialized in a client scenario to be used properly. This is less than ideal. 

We could further improve this by using an event listener called `device change` instead. More research needs to be done into this area. 

The device detection should be removed for now and merged in another PR, as is it currently just returns true every time, which also improves development if one does not have a device.

Reply:
resolved: https://github.com/Looking-Glass/looking-glass-webxr/pull/31/commits/95f72dbf99c787e9683681098c1c28023c7827d0